Makefile

```diff
@@ -82,12 +82,21 @@ ifneq ($(OVERRIDE_IMAGE_NAME),)
 endif
 	rm -rf $(TEMP_DIR)
 
-push: ./manifest-tool gcr-login $(addprefix sub-push-,$(ALL_ARCHITECTURES))
-	./manifest-tool push from-args --platforms $(ML_PLATFORMS) --template $(PREFIX)/heapster-ARCH:$(VERSION) --target $(PREFIX)/heapster:$(VERSION)
+do-push:
+	docker push $(PREFIX)/heapster-$(ARCH):$(VERSION)
+ifeq ($(ARCH),amd64)
```

Comment:
I did this for backwards compability since earlier versions are released as `heapster:${version}`

Grafana and influxdb completely changed image names, so it's safe to require the `-amd64` suffix there until gcr.io starts working as it should.